.github/workflows/single-runner.yml

```diff
@@ -4,13 +4,17 @@ on:
   workflow_dispatch:
     inputs:
       dataset:
-        description: 'Name of the dataset'
+        description: "Name of the dataset"
         required: true
         default: dcp_mappluto
       latest:
-        description: 'Tag this version as latest'
+        description: "Tag this version as latest"
         required: false
-        default: 'true'
+        default: "true"
+      version:
+        description: "Version of the dataset (i.e. 22v2, 20221121)"
```

Comment:
I wouldn't list `20221121` as an example here because that's not how this is intended to be used.  I'd change to
"Optional.  The version of the dataset (i.e. 22v2, 21C) if needed"

Reply:
Fixed